src/Store.lua

```diff
@@ -25,6 +25,8 @@ local Store = {}
 -- It may not be Heartbeat in the future.
 Store._flushEvent = RunService.Heartbeat
 
+Store.ACTION_LOG_LENGTH = 3
```

Comment:
Is this meant to be a property on an individual store, or just a pseudo-global that can be modified externally?